model/project_parser.go

```diff
@@ -705,6 +690,37 @@ func retrieveFile(ctx context.Context, opts GetProjectOpts) ([]byte, error) {
 	}
 }
 
+func retrieveFileForModule(ctx context.Context, opts GetProjectOpts, modules ModuleList, moduleName, path string) ([]byte, error) {
+	// Look through any given local modules first
+	if opts.LocalModules != nil {
+		if val, ok := (*opts.LocalModules)[moduleName]; ok {
+			moduleOpts := GetProjectOpts{
+				RemotePath:   fmt.Sprintf("%s/%s", val, path),
```

Comment:
val here actually is the path to module, right?

Reply:
yes, name clarification